pithos/plugins/notify.py

```diff
@@ -45,6 +45,14 @@ def on_prepare(self):
         if 'action-icons' in caps:
             self.notification.set_hint('action-icons', GLib.Variant('b', True))
 
+        # GNOME Shell 3.20 or higher has bultin MPRIS functionality that makes
+        # persistent song notifications redundant.
+        min_version = Gtk.get_major_version() >= 3 and Gtk.get_minor_version() >= 20
```

Comment:
> I didn't think it was possible to run a mismatched Gtk and GNOME Shell version?

Sure it is, especially with the previously mentioned Flatpak it will become more common.

> The server_info contains a version. But again that is the version of the server not necessarily the version of GNOME Shell.

The MPRIS server is in gnome-shell, so I think that will do.